dotcom-rendering/src/web/server/newsletters-page-model/defaultData.ts

```diff
@@ -0,0 +1,265 @@
+		},
+	],
+	otherLinks: [
+		{
+			title: 'The Guardian app',
+			url: 'https://www.theguardian.com/mobile/2014/may/29/the-guardian-for-mobile-and-tablet',
+		},
+		{ title: 'Video', url: '/video' },
+		{ title: 'Podcasts', url: '/podcasts' },
+		{ title: 'Pictures', url: '/inpictures' },
+		{ title: 'Newsletters', url: '/email-newsletters' },
+		{ title: "Today's paper", url: '/theguardian', children: [] },
+		{
+			title: 'Inside the Guardian',
+			url: 'https://www.theguardian.com/membership',
+		},
+		{ title: 'The Observer', url: '/observer', children: [] },
+		{
+			title: 'Guardian Weekly',
+			url: 'https://www.theguardian.com/weekly?INTCMP=gdnwb_mawns_editorial_gweekly_GW_TopNav_UK',
+		},
+		{ title: 'Crosswords', url: '/crosswords', children: [] },
+		{
+			title: 'Corrections',
+			url: '/theguardian/series/corrections-and-clarifications',
+		},
+	],
+	brandExtensions: [
+		{
+			title: 'Search jobs',
+			url: 'https://jobs.theguardian.com?INTCMP=jobs_uk_web_newheader_dropdown',
+		},
+		{
+			title: 'Hire with Guardian Jobs',
+			url: 'https://recruiters.theguardian.com/?utm_source=gdnwb&utm_medium=navbar&utm_campaign=Guardian_Navbar_Recruiters&CMP_TU=trdmkt&CMP_BUNIT=jobs',
+		},
+		{
+			title: 'Holidays',
+			url: 'https://holidays.theguardian.com?INTCMP=holidays_uk_web_newheader',
+		},
+		{
+			title: 'Live events',
+			url: 'https://membership.theguardian.com/events?INTCMP=live_uk_header_dropdown',
+		},
+		{ title: 'Masterclasses', url: '/guardian-masterclasses' },
+		{
+			title: 'Digital Archive',
+			url: 'https://theguardian.newspapers.com',
+		},
+		{
+			title: 'Guardian Print Shop',
+			url: '/artanddesign/series/gnm-print-sales',
+		},
+		{
+			title: 'Patrons',
+			url: 'https://patrons.theguardian.com/?INTCMP=header_patrons',
+		},
+		{
+			title: 'Guardian Puzzles app',
+			url: 'https://puzzles.theguardian.com/download',
+		},
+		{
+			title: 'Guardian content licensing site',
+			url: 'https://licensing.theguardian.com/',
+		},
+	],
+	readerRevenueLinks: {
+		header: {
+			contribute:
+				'https://support.theguardian.com/contribute?INTCMP=header_support_contribute&acquisitionData=%7B%22source%22:%22GUARDIAN_WEB%22,%22componentType%22:%22ACQUISITIONS_HEADER%22,%22componentId%22:%22header_support_contribute%22%7D',
+			subscribe:
+				'https://support.theguardian.com/subscribe?INTCMP=header_support_subscribe&acquisitionData=%7B%22source%22:%22GUARDIAN_WEB%22,%22componentType%22:%22ACQUISITIONS_HEADER%22,%22componentId%22:%22header_support_subscribe%22%7D',
+			support:
+				'https://support.theguardian.com?INTCMP=header_support&acquisitionData=%7B%22source%22:%22GUARDIAN_WEB%22,%22componentType%22:%22ACQUISITIONS_HEADER%22,%22componentId%22:%22header_support%22%7D',
+			supporter:
+				'https://support.theguardian.com/subscribe?INTCMP=header_supporter_cta&acquisitionData=%7B%22source%22:%22GUARDIAN_WEB%22,%22componentType%22:%22ACQUISITIONS_HEADER%22,%22componentId%22:%22header_supporter_cta%22%7D',
+		},
```

Comment:
I don't think we need 4 of them anymore, since we did launch the new supporter proposition.
Probably good to remove we don't support anymore.

Reply:
The fields are all still required as part of the `CAPINavType`, but I can use empty strings for the values - this module defines the sample data only used on the dev server for the '/EmailNewslettersTestData' route so just needs to match the shape of the data from frontend.